This patch allows mozilla to use the glibc-defined math functions on alpha,
instead of using the ones defined in mozilla's math library.  This gives us
a speed boost plus allows maps.google.com to actually work correctly!

NB: This patch is used for mozilla, mozilla-firefox and mozilla-thunderbird.

--- mozilla/js/src/jslibmath.h.agriffis	2005-07-11 14:58:35 -0400
+++ mozilla/js/src/jslibmath.h	2005-07-11 15:09:13 -0400
@@ -69,6 +69,9 @@
 #elif defined(HPUX)
 #define JS_USE_FDLIBM_MATH 1
 
+#elif defined(linux) && defined(__alpha__)
+#define JS_USE_FDLIBM_MATH 0
+
 #elif defined(linux)
 #define JS_USE_FDLIBM_MATH 1
 
